src/app/component/about/components/about-page/about-page.component.html

```diff
@@ -53,97 +54,87 @@ <h3 class="card-title-text">{{ 'about-us.steps.block-1.header' | translate }}</h
               <span class="main-card-text">
                 {{ 'about-us.steps.block-1.text' | translate }}
               </span>
-              <a class="card-link"
-                 [routerLink]="['/map']"
-                 fragment="top-user-bar"
-                 routerLinkActive="active-link"
-                 [routerLinkActiveOptions]="{ exact: true }">
+              <a class="card-link" [routerLink]="['/map']" fragment="top-user-bar" routerLinkActive="active-link"
```

Comment:
According to the style guide, when a tag has several attributes, it is more readable to write them one per line.

Reply:
Thanks, take it into account but this is not my code change.